spec.bs

```diff
@@ -310,7 +311,7 @@ This is detectable because it can change the set of fields that are read from th
               a {{TypeError}}.
           1. Set |igAd|'s [=interest group ad/allowed reporting origins=] to |allowedReportingOrigins|.
       1. [=list/Append=] |igAd| to |interestGroup|'s |interestGroupField|.
-1. If |interestGroup|'s [=interest group/estimated size=] is greater than 50 KB, then
+1. If |interestGroup|'s [=interest group/estimated size=] is greater than 1048576, then

```

Comment:
Is this number derived from the below section (`#interest-group-storage-maintenance`)? If so:

 1. Where is this number defined? `10*1024*1024` does not equal this number, it equals `10485760`, which is greater than this by a factor of 10.
 2. Can we also link to that section here as a pointer, for further reading (a note below this substep would be good).

Reply:
this is a different constant [kMaxInterestGroupSize](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/public/mojom/interest_group/interest_group_types.mojom;bpv=1;bpt=1;l=38?q=interest_group_types&ss=chromium&gsn=kMaxInterestGroupSize&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dmojom%23blink.mojom.kMaxInterestGroupSize) which is per IG size limit, different from the 10*1024*1024 [kInterestGroupStorageMaxStoragePerOwner](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/common/features.cc;drc=f5bdc89c7395ed24f1b8d196a3bdd6232d5bf771;bpv=1;bpt=1;l=976?gsn=kInterestGroupStorageMaxStoragePerOwner&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dthird_party%2Fblink%2Fcommon%2Ffeatures.cc%23pCLbr57w0v7pRWwLwx8KvKBfLg7yGadfO4Gcp922iec) which is for total IG size per owner.